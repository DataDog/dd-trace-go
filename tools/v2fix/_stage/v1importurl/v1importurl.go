// Unless explicitly stated otherwise all files in this repository are licensed
// under the Apache License Version 2.0.
// This product includes software developed at Datadog (https://www.datadoghq.com/).
// Copyright 2023 Datadog, Inc.

package main

import (
	"gopkg.in/DataDog/dd-trace-go.v1/ddtrace/ext"    // want `import URL needs to be updated`
	"gopkg.in/DataDog/dd-trace-go.v1/ddtrace/tracer" // want `import URL needs to be updated`
)

func main() {
	tracer.Start()
	defer tracer.Stop()

	span := tracer.StartSpan("operation", tracer.Tag(ext.SpanType, "test"))
	defer span.Finish()
}
