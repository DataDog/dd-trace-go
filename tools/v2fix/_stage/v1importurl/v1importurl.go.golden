-- update import URL to v2 --
// Unless explicitly stated otherwise all files in this repository are licensed
// under the Apache License Version 2.0.
// This product includes software developed at Datadog (https://www.datadoghq.com/).
// Copyright 2023 Datadog, Inc.

package main

import (
	"github.com/DataDog/dd-trace-go/v2/ddtrace/ext"    // want `import URL needs to be updated`
	"github.com/DataDog/dd-trace-go/v2/ddtrace/tracer" // want `import URL needs to be updated`
)

func main() {
	tracer.Start()
	defer tracer.Stop()

	span := tracer.StartSpan("operation", tracer.Tag(ext.SpanType, "test"))
	defer span.Finish()
}
